lib/mappers/figma/color.js

```diff
@@ -0,0 +1,7 @@
+const { mapColorKeys, formatColor } = require('../../utils');
+
+const getColorProperty = component => component.children[0].fills[0].color;
```

Comment:
Why put this in a separate function when you can just provide it as an argument? I see no reuse and no reason to put it in its own function.